UnitsNet.Tests/GeneratedCode/TestsBase/AccelerationTestsBase.g.cs

```diff
@@ -397,378 +397,78 @@ public void ToUnit_UnitSystem_ThrowsArgumentExceptionIfNotSupported()
-                AssertEx.EqualTolerance(1, parsed.MicrometersPerSecondSquared, MicrometersPerSecondSquaredTolerance);
-                Assert.Equal(AccelerationUnit.MicrometerPerSecondSquared, parsed.Unit);
-            }
-
-            {
-                Assert.True(Acceleration.TryParse("1 mm/s²", CultureInfo.GetCultureInfo("en-US"), out var parsed));
-                AssertEx.EqualTolerance(1, parsed.MillimetersPerSecondSquared, MillimetersPerSecondSquaredTolerance);
-                Assert.Equal(AccelerationUnit.MillimeterPerSecondSquared, parsed.Unit);
-            }
-
-            {
-                Assert.True(Acceleration.TryParse("1 мм/с²", CultureInfo.GetCultureInfo("ru-RU"), out var parsed));
-                AssertEx.EqualTolerance(1, parsed.MillimetersPerSecondSquared, MillimetersPerSecondSquaredTolerance);
-                Assert.Equal(AccelerationUnit.MillimeterPerSecondSquared, parsed.Unit);
-            }
-
-            {
-                Assert.True(Acceleration.TryParse("1 mg", CultureInfo.GetCultureInfo("en-US"), out var parsed));
-                AssertEx.EqualTolerance(1, parsed.MillistandardGravity, MillistandardGravityTolerance);
-                Assert.Equal(AccelerationUnit.MillistandardGravity, parsed.Unit);
-            }
-
-            {
-                Assert.True(Acceleration.TryParse("1 мg", CultureInfo.GetCultureInfo("ru-RU"), out var parsed));
-                AssertEx.EqualTolerance(1, parsed.MillistandardGravity, MillistandardGravityTolerance);
-                Assert.Equal(AccelerationUnit.MillistandardGravity, parsed.Unit);
-            }
-
-            {
-                Assert.True(Acceleration.TryParse("1 nm/s²", CultureInfo.GetCultureInfo("en-US"), out var parsed));
-                AssertEx.EqualTolerance(1, parsed.NanometersPerSecondSquared, NanometersPerSecondSquaredTolerance);
-                Assert.Equal(AccelerationUnit.NanometerPerSecondSquared, parsed.Unit);
-            }
-
-            {
-                Assert.True(Acceleration.TryParse("1 нм/с²", CultureInfo.GetCultureInfo("ru-RU"), out var parsed));
-                AssertEx.EqualTolerance(1, parsed.NanometersPerSecondSquared, NanometersPerSecondSquaredTolerance);
-                Assert.Equal(AccelerationUnit.NanometerPerSecondSquared, parsed.Unit);
-            }
-
-            {
-                Assert.True(Acceleration.TryParse("1 g", CultureInfo.GetCultureInfo("en-US"), out var parsed));
-                AssertEx.EqualTolerance(1, parsed.StandardGravity, StandardGravityTolerance);
-                Assert.Equal(AccelerationUnit.StandardGravity, parsed.Unit);
-            }
-
-            {
-                Assert.True(Acceleration.TryParse("1 g", CultureInfo.GetCultureInfo("ru-RU"), out var parsed));
-                AssertEx.EqualTolerance(1, parsed.StandardGravity, StandardGravityTolerance);
-                Assert.Equal(AccelerationUnit.StandardGravity, parsed.Unit);
-            }
```

Comment:
Man, we do have some verbose tests 😓 Can't say I browse and read these files more than I absolutely have to, but at least it's getting better!